modules/shared/components/molecules/TabGroup/Tab.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import classNames from 'classnames';
+import type { FC, ReactElement } from 'react';
+import styles from './TabGroup.module.css';
+import Radio from '../../atoms/Radio/index';
+import type { ITabGroup } from './types/ITabGroup';
+
+const Tab: FC<ITabGroup.ITabProps> = (props): ReactElement => {
+  const {
+    value,
+    id,
+    svg,
+    children,
+    checkedValue = 'Image Poll',
+    changeValHandler,
+    disabled,
+    onlyLabel,
+  } = props;
+
+  const tabClasses: string = classNames(styles['tab-default'], {
+    [styles['tab-checked']]: value === checkedValue,
+    [styles['tab-unchecked']]: value !== checkedValue,
+    [styles['tab-disabled']]: disabled,
+  });
+
+  return (
+    <label
+      htmlFor={id}
+      data-value={value}

```

Comment:
What the use for this attribute?

Reply:
It's for future testing purposes when we lift up the state to the parent component ,
Check out the pr description